### DISCUSSION TITLE
skypeforlinux: deny access to the Home directory and subfolders

### DISCUSSION THREAD

**Discussion by yderi** [netblue30/firejail#4618] [Q&A]
Hello everyone

Currently in use:
firejail version 0.9.66
skypeforlinux version: 8.77
Ubuntu 21.10

Current "skypeforlinux.profile" allows sneaking the Home folder and all other subfolders that are not a system like Video, Pictures, Music etc. 
If I comment out the line "ignore whitelist ${DOWNLOADS}" it limits access to Downloads only, but requires authentication every time you quit the Skype.

How to restrict to ~/Downloads only (and other relative stuff that it may require)?

Will appreciate any help on this.

**Comment by rusty-snake** [comment]
Actually it would be the best if you could try to use (i.e. remove the line) this hole block: 
https://github.com/netblue30/firejail/blob/57b9ff0717e96ce46041da008b2e1a2b5a472591/etc/profile-m-z/skypeforlinux.profile#L8-L18

- Instead of `dbus-user none`, use the policy from #4494.
- `novideo` is oblivious wrong.
- `nou2f` maybe too.
- Regarding `whitelist ${DOWNLODAS}`, also use `include whitelist-common.inc` and follow these steps
  (or just look for `~/.skype*`, `~/.config/skype*` ...)
  https://github.com/netblue30/firejail/blob/57b9ff0717e96ce46041da008b2e1a2b5a472591/etc/templates/profile.template#L62-L73
- wusc (whitelist-usr-share-common) might need `whitelist /usr/share/skype` or so in addition.
